realm/realm-library/src/androidTest/java/io/realm/internal/CollectionTests.java

```diff
@@ -258,7 +260,7 @@ public void addListener_shouldBeCalledToReturnTheQueryResults() {
         looperThread.keepStrongReference(collection);
         collection.addListener(collection, new RealmChangeListener<Collection>() {
             @Override
-            public void onChange(Collection collection1) {
+            public void onChange(@Nonnull Collection collection1) {
```

Comment:
I've updated this PR.

I removed each `@Nonnull` annotation from overriding methods and added more `package-info` instead.